src/components/common/Dialogs/EditColonyDetailsDialog/EditColonyDetailsDialogForm.tsx

```diff
@@ -0,0 +1,222 @@
+import React, { useState } from 'react';
+import { FormattedMessage, defineMessages } from 'react-intl';
+import { ColonyRole, Id } from '@colony/colony-js';
+import { useFormContext } from 'react-hook-form';
+
+import AvatarUploader from '~shared/AvatarUploader';
+import {
+  ActionDialogProps,
+  DialogControls,
+  DialogHeading,
+  DialogSection,
+} from '~shared/Dialog';
+import {
+  Annotations,
+  HookFormInput as Input,
+  InputStatus,
+} from '~shared/Fields';
+import ColonyAvatar from '~shared/ColonyAvatar';
+// import NotEnoughReputation from '~dashboard/NotEnoughReputation';
+import Avatar from '~shared/Avatar';
+import { useActionDialogStatus } from '~hooks';
+
+import {
+  CannotCreateMotionMessage,
+  NoPermissionMessage,
+  PermissionRequiredInfo,
+} from '../Messages';
+
+import styles from './EditColonyDetailsDialogForm.css';
+
+const displayName =
+  'common.EditColonyDetailsDialog.EditColonyDetailsDialogForm';
+
+const MSG = defineMessages({
+  title: {
+    id: `${displayName}.title`,
+    defaultMessage: 'Edit colony details',
+  },
+  name: {
+    id: `${displayName}.name`,
+    defaultMessage: 'Colony name',
+  },
+  logo: {
+    id: `${displayName}.logo`,
+    defaultMessage: 'Colony Logo (Optional)',
+  },
+  permittedFormat: {
+    id: `${displayName}.permittedFormat`,
+    defaultMessage: 'Permitted format: .png or .svg',
```

Comment:
If the colony avatar upload is handled the same way as the user avatar, showing this message will not be necessary as the file type will not be limited to png and svg.

Reply:
Yeah, that will probably be the case.

Obliterating this message :gun: